### ISSUE TITLE
Edit contact information on complaints 

### ISSUE THREAD

**Issue by lalitha-jonnalagadda** [18F/crt-portal#431]
## Narrative
As an intake specialist, I would like to edit the contact information on a complaint after its created so that I can correct or update the latest and greatest contact details 

## Background
Upon receiving a complaint, sections often contact the complainant for additional information, which would require the intake specialists to have the ability to edit the original complaint to add new information. At time, complainants reach out to the sections to update their contact information so sections have the most recent data

## Assumptions
- All changes/edits will be captured in the activity log
- All fields in the contact section can be edited at once

## Out of scope
- Editing the reported complaint

## Acceptance Criteria
1. Given that I'm on the complaint details page
When I am in the Contact information section and click on the first name, last name, address, email or phone number field
Then I see a pencil icon labelled 'edit' 
AND 
When I click on the 'edit' icon
Then I see an option to edit as well as a 'Save' or 'Cancel' option 

2. Given that I clicked on the 'edit' option 
When start typing information 
Then I see the previous information replaced with the new information  
AND
I'm able to save my edits by clicking on the 'Save' button

3. Given that I edited contact information on a complaint
When I check the activity panel for that complaint
Then I see a log for the edits on the complaint 

4. a11y ( @Jacklynn please help!)

## Mock ups
Invision Mock - https://gsa.invisionapp.com/share/X5WDHCYW49U#/screens/408944113

## Sign off
- [x] PO
UAT 
- [x] PO
- [ ] Product Design
- [ ] a11y

**Comment by Jacklynn** [comment]
Could we align the text more horizontally and with the other edit button?
**Current:** 
![Screen Shot 2020-04-24 at 4.45.35 PM.png](https://images.zenhubusercontent.com/5d1a3b0bcc794602e102a513/0a2ddf64-494a-49e7-9477-c8714504acb5)

Expected:
![Screen Shot 2020-04-24 at 4.48.58 PM.png](https://images.zenhubusercontent.com/5d1a3b0bcc794602e102a513/602024ed-3f98-41f2-85d9-7e871438579d)

**Comment by Jacklynn** [comment]
**A11y**
With Safari + VO, I typed and incorrect email and hit "save." The page refreshed but I didn't hear a notification that it didn't save or that the email field needed fixing. If I navigated to the field it'll say "invalid entry" but i would have to find it. 

**Another design thing**
low priority, but the error message is missing the icon that's on the front end. 
![Screen Shot 2020-04-24 at 4.54.47 PM.png](https://images.zenhubusercontent.com/5d1a3b0bcc794602e102a513/0cacb8f4-0941-43f2-97d4-f7d1d709abcf)
